spec/controllers/whitehall_assets_controller_spec.rb

```diff
@@ -94,6 +94,17 @@
         expect(response).to have_http_status(:unprocessable_entity)
       end
     end
+
+    context "an asset with the same legacy_url_path as an existing asset" do
+      let(:attributes) { FactoryBot.attributes_for(:whitehall_asset, :with_legacy_metadata) }
```

Comment:
Minor: Could you move the `let(:attributes)` statement into the outer context to avoid duplicating it here?

Reply:
I'll make that change.